llvm/test/CodeGen/X86/isel-select-cmov.ll

```diff
@@ -207,6 +219,13 @@ define zeroext i16 @select_cmov_i16(i1 zeroext %cond, i16 zeroext %a, i16 zeroex
 ; GISEL-X86-CMOV-NEXT:    cmovnew %dx, %ax
 ; GISEL-X86-CMOV-NEXT:    ## kill: def $ax killed $ax killed $eax
 ; GISEL-X86-CMOV-NEXT:    retl
+;
+; NDD-LABEL: select_cmov_i16:
+; NDD:       ## %bb.0:
```

Comment:
Isn't it a bug here? CMOV is not enabled but you CMOV is generated.

Reply:
Because from source code, HasCMOV is rewritten, we could use it in other cases.
`def HasCMOV      : Predicate<"Subtarget->canUseCMOV()">;`
`bool canUseCMOV() const { return hasCMOV() || hasSSE1() || is64Bit(); }`